Cacti-SSL-Configuration.md

```diff
@@ -11,19 +13,23 @@ cp ca.crt /etc/pki/tls/certs
 cp ca.key /etc/pki/tls/private/ca.key
 cp ca.csr /etc/pki/tls/private/ca.csr
 
+```
+
 Then we need to update the Apache SSL configuration file:
 
+```ini
 vi +/SSLCertificateFile /etc/httpd/conf.d/ssl.conf
 SSLCertificateFile /etc/pki/tls/certs/ca.crt
 SSLCertificateKeyFile /etc/pki/tls/private/ca.key
+```
 
 Restart the httpd service:
 
 systemctl restart httpd
 
 After configuring the web server to accept https, you can enable https in the GUI
 
-## Note, if using multiple pollers
+### Note, if using multiple pollers
 
 all must have HTTPS enabled for the remote polling feature to work properly.
```

Comment:
I think we should update this to something like 

# Remote Poller Setup

When using remote pollers, you MUST enable SSL between the main and remote pollers.  Cacti will not function without secure communications.